src/Mutator/FunctionSignature/ProtectedVisibility.php

```diff
@@ -65,6 +65,11 @@ private function hasSameProtectedParentMethod(Node $node): bool
         /** @var \ReflectionClass $reflection */
         $reflection = $node->getAttribute(ReflectionVisitor::REFLECTION_CLASS_KEY);
 
+        if (!$reflection instanceof \ReflectionClass) {
+            // assuming the worst where interface has the same method
+            return true;
+        }
```

Comment:
PHPStan says that a strict comparison using `===` between `ReflectionClass` and `null` will always evaluate to false.

Reply:
I think this is because you have `/** @var \ReflectionClass $reflection */` instead of `/** @var \ReflectionClass|null $reflection */`.

I'm not against the instanceof check, however I admit I would prefer `!($reflection instanceof \ReflectionClass)`. That said it's personal preference over CS so pay no mind to it if you disagree

Reply:
I'm not the one to disagree! This is definitely more proper because leaves no window for error.